### DISCUSSION TITLE
Energy Total (kWh meter) only full kWh increments?

### DISCUSSION THREAD

**Discussion by Thyraz** [marq24/ha-goecharger-api2#16] [General]
Hi,

is it normal behavior, that the total energy sensor only increases by full 1 kWh steps?
The "energy since car connected" sensor is far more fine-grained, so I'm not sure if the go-e doesn't provide a better value, or if there's something wrong with my configuration.

Thanks in advance,
Tobias

**Comment by marq24** [comment]
no - looks like something is wrong with my coding skills! - Thanks for making this aware to me (a fix is just around the corner)!

**Reply by Thyraz** [reply]
Thanks a lot. 👍

**Comment by marq24** [comment]
fixed in https://github.com/marq24/ha-goecharger-api2/releases/tag/2024.9.1

**Reply by Thyraz** [reply]
I updated yesterday and just connected the car for a test.
Works perfectly. Thank you very much for the quick fix. 🙂👍
